examples/plot_kernel_approximation.py

```diff
@@ -68,12 +68,14 @@
 data -= data.mean(axis=0)
 
 # We learn the digits on the first half of the digits
-data_train, targets_train = data[:n_samples / 2], digits.target[:n_samples / 2]
+data_train, targets_train = (data[:int(n_samples / 2)],
+                             digits.target[:int(n_samples / 2)])
 
 
 # Now predict the value of the digit on the second half:
-data_test, targets_test = data[n_samples / 2:], digits.target[n_samples / 2:]
-#data_test = scaler.transform(data_test)
+data_test, targets_test = (data[int(n_samples / 2):],
```

Comment:
depending on whether n_samples is even or not this might actually be a line that raises a warning. But why not just use integer division?